### ISSUE TITLE
Imports are not being ordered correctly.

### ISSUE THREAD

**Issue by abrougher** [snowcoders/sortier#444]
# Language
Typescript

# Sample input source code
import { Po, IP } from '@foo';

# Expected output
import { IP, Po } from '@foo';

# Actual output (or error message)
import { Po, IP } from '@foo';

In this example 'Po' and 'IP' are not being sorted correctly. It seems to be dependent on the second char? Everything else seems to be sorted correctly. I'm only seeing issues when sorting imports. 

I expected the sort to satisfy tslint's ordered-imports.

**Comment by k2snowman69** [comment]
By default specifiers are sorted by ["*", "interfaces", "types"] where an interface is defined based on a capital I followed by a capital letter, for example "IPlayer" or "IFactory" (see [code](https://github.com/snowcoders/sortier/blob/master/src/language-js/sortImportDeclarationSpecifiers/index.ts#L136)).

The code to allow for overrides has been written (see the code above) it's just not being passed through the layers (whoops). After the code is fixed, the sortier config file to override the default specifier sort would look something like this:
```
{
  ...configs.sortier,
  js: {
    ...configs.sortier.js,
    sortImportDeclarationSpecifiers: {
      groups: ["*"]
    }
}
```

**Comment by abrougher** [comment]
Amazing! Thanks for the quick response!

**Comment by k2snowman69** [comment]
I have about 30 minutes today where I can play with it, I'll push up a new branch and update this comment with the branch information if it doesn't get fixed. If it does get fixed, I'll add you to the PR for review :-)

**Comment by abrougher** [comment]
Thanks!  Even used my example. LGTM!

Should I close the issue?

**Comment by k2snowman69** [comment]
Not yet, I'll close it once the new version of the npm package is released along with the version it was fixed in

**Comment by k2snowman69** [comment]
Just released 2.6.2, should be fixed with that. I'm going to close this issue; let me know if you still run into issues!
